pkg/controller/ovn-ic.go

```diff
@@ -146,6 +147,13 @@ func (c *Controller) establishInterConnection(config map[string]string) error {
 			blackList = append(blackList, subnet.Spec.CIDRBlock)
 		}
 	}
+	nodes, err := c.nodesLister.List(labels.Everything())
```

Comment:
node and subnet may changed, blacklist set should be outside the `establishInterConnection`